misaligned.c

```diff
@@ -4,12 +4,14 @@
 int printColorMap() {
     const char* majorColor[] = {"White", "Red", "Black", "Yellow", "Violet"};
     const char* minorColor[] = {"Blue", "Orange", "Green", "Brown", "Slate"};
-    int i = 0, j = 0;
+    int i = 0, j = 0, k = 0;
     for(i = 0; i < 5; i++) {
         for(j = 0; j < 5; j++) {
-            printf("%d | %s | %s\n", i * 5 + j, majorColor[i], minorColor[i]);
+            k = i*5+j;
+            printf("%d | %s | %s\n", k, majorColor[i], minorColor[i]);
         }
     }
+    assert(k==(i*j));

```

Comment:
Test cases need to be enhanced to check the alignment of the output text. Also keep in mind that there are single digit pair numbers and double digit pair numbers too.